src/main/java/com/dataloom/datastore/permissions/controllers/PermissionsController.java

```diff
@@ -44,7 +44,7 @@
     @RequestMapping(
```

Comment:
hm, I think that is the case? Or do you mean I should put the controller path up top rather than setting the absolute path at each endpoint?